docs/man/man1/kubectl-drain.1

```diff
@@ -0,0 +1,154 @@
+.TH "KUBERNETES" "1" " kubernetes User Manuals" "Eric Paris" "Jan 2015"  ""
+
+
+.SH NAME
+.PP
+kubectl drain \- Drain nodes in preparation for maintenance
+
+
+.SH SYNOPSIS
+.PP
+\fBkubectl drain\fP [OPTIONS]
+
+
+.SH DESCRIPTION
+.PP
+Drain node in preparation for maintenance.
+
+.PP
+The given node will be marked unschedulable to prevent new pods from arriving.
+Then all running pods will be removed from the machine.  If there are any
+unreplicated pods, no pods will be touched without \-\-force.
```

Comment:
It looks like when you changed the meaning of omitting --force from "skip unreplicated pods" to "skip the whole machine if there are any unreplicated pods" you forgot to update some of the descriptions. So for example while the above is correct, the description a few lines later under OPTIONS is incorrect, and likewise for the comment on the second EXAMPLE at the end of this file. Apply the same to kubectl_drain.md.